packages/frontend-shared/src/locales/en-US.json

```
@@ -272,7 +272,7 @@
           "noParallelTime": "12 mins",
           "withParallelization": "with Parallelization",
           "withoutParallelization": "without Parallelization",
-          "intro": "With Smart Orchestration, you'll be able to:",
+          "intro": "With smart orchestration, you'll be able to:",
```

Although not noted explicitly in the style guide, this is a named Cypress feature and therefore a proper noun

```suggestion
          "intro": "With Smart Orchestration, you'll be able to:",
```